bin/cluster

```diff
@@ -290,6 +291,8 @@ if __name__ == '__main__':
                                help='number of infra nodes to create in cluster')
     create_parser.add_argument('-e', '--etcd', default=0, type=int,
                                help='number of external etcd hosts to create in cluster')
+    create_parser.add_argument('--env', default='prod', type=str,
```

Comment:
I understand that its beyond our internal usage.  int/stg/prod are all suggestions here.  Could use dev?  test?  Any other suggestions?